backend/yugiohcardset/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+from yugiohcardinset.models import YugiohCardInSet
+
+
+class YugiohCardSet(models.Model):
+    card_set_name = models.CharField(
+        max_length=100,
+        unique=True,
+    )
```

Comment:
- [ ] Add `code` field

The code is unique. Exemplary code and set would be: `Phantom Nightmare` - `PHNI`

Reply:
done